.gitlab-ci.yml

```diff
@@ -25,7 +25,7 @@ build:
       - dist/*.zip
```

Comment:
do we need to even keep the gitlab flow? this might be a good time to remove it altogether. actions seem like a better solution to me.

Reply:
I feel like it's not hurting anything but I'll remove it 🤷